src/index.js

```diff
@@ -1,11 +1,30 @@
 import React from "react";
 import ReactDOM from "react-dom";
+import "bootstrap/dist/css/bootstrap.min.css";
 import "./index.css";
 import App from "./App";
+import { Provider } from "react-redux";
+import { createStore, applyMiddleware, compose } from "redux";
+import createSagaMiddleware from "redux-saga";
+import { regisRequest } from "./saga/Registrationsaga";
+import rootReducer from "../src/Redux/reducers/combinereducer";
+
+const sagaMiddleware = createSagaMiddleware();
+
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
```

Comment:
import { applyMiddleware, createStore } from "redux";

use applyMiddleware of redux instead of __REDUX_DEVTOOLS_EXTENSION_COMPOSE__

Reply:
no need of __REDUX_DEVTOOLS_EXTENSION_COMPOSE__

Reply:
i have used {applyMiddleware} for redux-saga
const store = createStore(
  rootReducer,
  composeEnhancers(applyMiddleware(sagaMiddleware))
);
----REDUX_DEVTOOLS_EXTENSION_COMPOSE 
this is used for using redux dev tool

Reply:
ok